docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -96,7 +96,7 @@ These templates are loaded when the integration is installed, and are used to co
 
 [discrete]
 [[data-streams-ilm]]
-== {ilm} ({ilm-init})
+= {ilm} ({ilm-init})
```

Comment:
Can we change this while we're here? This is bugging me. Sorry, I can't upload a screenshot for some reason. It's a capitalization issue.

```suggestion
= Index lifecycle management ({ilm-init})
```